_site/index.html

```diff
@@ -547,6 +588,36 @@ <h3>T-Z</h3>
           </div>
         </section>
 
+        <section>
+          <p class="word__title" id="non-binary">
+            Non-Binary
+            <span class="word__speech">
+              adj
+            </span>
+          </p>
+          
+          <p class="word__definition">
+            someone whose gender does not fit into the male-female gender binary.<br><br>
+            
+            For example, a non-binary person may have some gender between male and female, have multiple genders, no gender, a culturally-specific gender (e.g. two-spirit).
+          
+          </p>
+
+          <div class="block__type">
+            <p class="word__type">
+              Note
+            </p>
+            <p class="word__breakdown">
+              <span class="word__alt">
+                Non-binary people may or may not also identify as <a class="word__link" href="#transgender">transgender</a>.<br><br>
+
+                Although non-binary is often used as an umbrella term for multiple gender identities, some people who fall under that umbrella may not themselves use the descriptor "non-binary".
```

Comment:
This section is a little vague to me. I wonder if it helps to reframe it a bit?

"Non-binary can be used as an umbrella term for people with many types of gender identities (e.g., `xxxxxx`). However, note that just because someone identifies as `xxxxxx`, does not necessarily imply that they identify as non-binary as well."

Reply:
I reframed it by moving the "umbrella term" part into the main definition, then providing the "however" and example in the notes section.